tests/test_cli_v1.py

```diff
@@ -76,7 +78,11 @@ def test_package_upgrade(name, args):
 
         expected_root = Path(root + '-expected')
         actual_root = Path(tmp_dir)
-        expected = _load_tree(expected_root)
+        expected = _load_tree(
+            expected_root,
+            # Cope with posix style reference data on Windows
+            replace_name=str(expected_root).replace(os.path.sep, '/'),
+        )
         actual = _load_tree(actual_root)
 
         assert actual == expected
```

Comment:
Path separators in the output are platform-specific too. I think it's safe for testing purposes to normalize all `\` to `/` in the `actual` output.